packages/salesforcedx-vscode-core/package.json

```diff
@@ -57,7 +57,10 @@
     "test":
       "cross-env CODE_TESTS_WORKSPACE='../system-tests/assets/sfdx-simple' node ./node_modules/vscode/bin/test"
   },
-  "activationEvents": ["workspaceContains:sfdx-project.json"],
+  "activationEvents": [
+    "workspaceContains:sfdx-project.json,",
+    "onCommand:sfdx.force.project.create"
```

Comment:
Adding an additional activation event for the extension to allow the create command to execute. Two scenarios arise from this:
1) Project create command is ran but does not go execute (i.e. user cancels out of input), all other commands are still hidden because of their when clause.
2) Project create command is ran and executes all the way through causing the host to refresh with the new folder and go through the extension loading again. Since the command creates sfdx project, sfdx-project.json will be present and the extension will activate, showing all the commands.